backend/modules/auth/router.py

```
@@ -1,33 +1,656 @@
+used_authorization_codes = set()
+
+
+def add_cors_headers(response: Response):
+    """Add CORS headers for cookie support"""
+    response.headers["Access-Control-Allow-Credentials"] = "true"
+    response.headers["Access-Control-Allow-Origin"] = FRONTEND_URL
+
+
+def create_oauth_success_redirect(user_login_response, access_token: str) -> RedirectResponse:
+    """Create proper HTTP redirect for successful OAuth"""
+    import json
```

Ruff (0.14.6)

Redefinition of unused `json` from line 8

Remove definition: `json`

(F811)